tests/test_data.py

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+test_data
+Test for Dataset class 
+@author: jdiedrichsen
+"""
+
+import unittest 
+import pyrsa.data as rsd
+import numpy as np 
+
+class TestSimulation(unittest.TestCase): 
```

Comment:
Is this for the Simulation package? It does not seem to test RDM code?

Reply:
I haven't implemented the test part yet for the rdm classes. I plan to do that as a next step (perhaps in the next pull), or do you suggest me to provide the test now in this pull?

Reply:
I will add test_rdm.py in my next few pushes. Thanks for noting.

Reply:
fixed and pushed.